tests/helpers_tests/test_condition.py

```diff
@@ -17,117 +17,342 @@
 
 from tests import base
 
+browserConditionSafari = ['browser_type', 'safari', 'custom_attribute', 'exact']
+booleanCondition = ['is_firefox', True, 'custom_attribute', 'exact']
+integerCondition = ['num_users', 10, 'custom_attribute', 'exact']
+doubleCondition = ['pi_value', 3.14, 'custom_attribute', 'exact']
 
-class ConditionEvaluatorTests(base.BaseTest):
+exists_condition_list = [['input_value', None, 'custom_attribute', 'exists']]
+exact_string_condition_list = [['favorite_constellation', 'Lacerta', 'custom_attribute', 'exact']]
+exact_number_condition_list = [['lasers_count', 9000, 'custom_attribute', 'exact']]
```

Comment:
Can we also have a test for exact number match where the condition value is a float? I see the `doubleCondition` above, but it looks like that's used for something else